app/uk/gov/hmrc/customs/declaration/services/UploadedFileProcessingService.scala

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.customs.declaration.services
+
+import java.util.UUID
+
+import com.google.inject.{Inject, Singleton}
+import uk.gov.hmrc.customs.declaration.connectors.CustomsNotificationServiceConnector
+import uk.gov.hmrc.customs.declaration.controllers.{FileStatus, UpscanNotification}
+
+import scala.concurrent.Future
+import scala.xml.NodeSeq
+
+
+case class UploadedFileDetails(
```

Comment:
Some of these parameters seem to be unused eg `eori` - this may be because they will be used in next PR - if not pleasde remove. Also consider re-using existing value types such as `Eori `. Currently this endpoint code is deviating from the existing pattern of of using value types in the rest of end points, so there is an inconstancy there

Reply:
We are receiving the EORI but This story does not deal with the EORI, hence no attention to this parameter